lab_7_llm/main.py

```diff
@@ -41,6 +46,7 @@ def obtain(self) -> None:
         Raises:
             TypeError: In case of downloaded dataset is not pd.DataFrame
         """
+        self._raw_data = DataFrame(load_dataset(self._hf_name, split='if_test'))
```

Comment:
Use the method `to_pandas` after calling `load_dataset`, as such an implementation will be cleaner.